pkg/debugapi/peer.go

```diff
@@ -35,6 +35,14 @@ func (s *server) peerConnectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	s.Addressbook.Put(address, addr)
+	if err := s.TopologyDriver.AddPeer(address); err != nil {
```

Comment:
i wonder if we should add the flag forcing the connection to the `AddPeer` signature already. this is relevant to the debug api behavior. although it is true that right now this functionality will not really do anything, so, at your discretion

Reply:
Again, related to the stuff above, depends if we want to expose multiaddr on topology interface or just implementation, it was related to the same decision. I took the liberty to do it a bit opinionated, but now when we see it, we can change if you guys think it's better the other way.